### DISCUSSION TITLE
Auth problem

### DISCUSSION THREAD

**Discussion by moinuddinrepos** [community/community#137769] [Code Security] [answered]
### Select Topic Area

General

### Body

I have old GitHub account it secured with auth code. Problem is i am not getting any auth code or scanner even i don't remember a security answers. How to get back my old account basically there is no way to get back it. If anyone knows let me know Thank you.

- This is my old GitHub id: @moinuddin7866

**Comment by LiteBrite82** [comment, marked as answer]
Hi @moinuddinrepos ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
